### DISCUSSION TITLE
Is it possible to make NvChad exit automatically after first run

### DISCUSSION THREAD

**Discussion by hexdigest** [NvChad/NvChad#2131] [Q&A] [answered]
The reason why I'm asking is because I'm trying to automate a deployment of my work setup on fresh Ubuntu installation which obviously include neovim and nvchad. The problem is that the process interrupts on:

> Do you want to install example of custom config? (y/N):

And

```
git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim
```

When I just have to exit nvim (which is itself anecdotal)

**Comment by siduck** [comment]
@hexdigest try nvim --headless

**Reply by hexdigest** [reply, marked as answer]
Thanks for the suggestion @siduck !

It doesn't run editor itself but the dialog remains and stops my installation script and a CI docker build.
I found similar problems with packer:

https://github.com/wbthomason/packer.nvim/pull/109 may be some similar solution can be considered for nvchad

Another alternative that could work for me personally is that if nvchad detects that nvim is in headless mode it can just apply defaults (N), because I'm symlinking my custom configs after anyways.

**Comment by hexdigest** [comment]
Яя
